cmake/Common/CompilerArmClangCommon.cmake

```diff
@@ -184,13 +184,13 @@ function(compiler_merge_library)
 	target_sources(${_MY_PARAMS_DEST} PRIVATE ${_MY_PARAMS_LIBS})
 endfunction()
 
-function(compiler_generate_binary_output TARGET)
+function(compiler_generate_binary_output TARGET DIR)
```

Comment:
if doing this the correct way, as part of install target, see `<link>`, then this `DIR` patch shouldn't be needed